tests/MessageBundlesCompilerTests.js

```diff
@@ -19,7 +19,7 @@ var gpii = fluid.registerNamespace("gpii");
 
 var jqUnit = fluid.require("node-jqunit", require, "jqUnit");
 
-require("../messageBundlesCompiler.js");
+require("../scripts/shared/messageBundlesCompiler.js");
```

Comment:
Changing to `fluid.require("%gpii-app/scripts/shared/messageBundlesCompiler")` might help

Reply:
Nice catch, I didn't realize because I ran the affected test only (without coverage reporting). I've updated it following your suggestion.